src/services/httpBaseService.js

```diff
@@ -0,0 +1,26 @@
+const BASE_URL = ' https://pokeapi.co/api/v2/';
```

Comment:
Good abstractions here for the two methods, though I think options are not necessary needed.